src/DGtal/geometry/doc/moduleDigitalConvexityApplications.dox

```diff
@@ -206,5 +207,53 @@ how it is computed internally:
 
 \image html cps-shortest-path.jpg "Shortest path between two points" width=500
 
+@subsection dgtal_dconvexityapp_sec25 Speed-up shortest paths with some approximation
+
+Classes TangencyComputer and TangencyComputer::ShortestPaths offer a
+way to speed up considerably shortest paths computation if one
+tolerates a little bit of approximation. Several methods accept a
+distance parameter \a K. If \a K is greater than \f$
+\sqrt{d} \f$, then shortest paths are exact, but if you give a lower
```

Comment:
it is "exact" in the sense that it produces the same results as in the previous section from tangency predicate + graph based A*.
The "exact" is bit misleading to me

Reply:
Yes. Or equivalently, the result is a shortest path. I reformulate my sentence.